google/cloud/bigquery/v2/minimal/internal/job.h

```diff
@@ -44,7 +44,7 @@ void from_json(nlohmann::json const& j, JobStatus& jb);
 struct JobReference {
   std::string project_id;
   std::string job_id;
-  std::string location;
```

Comment:
It may be better to leave this empty and get an error than to fill-in `"US"` and surprise a caller with the wrong behavior.